src/c_types.py

```diff
@@ -368,6 +369,7 @@ def parse_enum(enum: ca.Enum, typemap: TypeMap) -> None:
             value = next_value
         next_value = value + 1
         typemap.enum_values[enumerator.name] = value
+        typemap_enum.values[value] = enumerator.name
```

Comment:
So if there are multiple names mapping to a single value we take the last one. Makes sense; maybe worth observing in a comment